src/lib/radio/radio.ts

```diff
@@ -129,8 +137,19 @@ export class MdRadioGroup extends _MdRadioGroupMixinBase
     this.labelPosition = (v == 'start') ? 'after' : 'before';
   }
 
+
   /** Whether the labels should appear after or before the radio-buttons. Defaults to 'after' */
-  @Input() labelPosition: 'before' | 'after' = 'after';
+  @Input()
+  get labelPosition() {
+    return this._labelPosition;
+  }
+
+  set labelPosition(v) {
+    this._labelPosition = (v == 'before') ? 'before' : 'after';
+    if (this._radios) {
+      this._radios.forEach(radio => radio._groupValueChanged());
+    }
```

Comment:
I think this part should be broken out into its own function
```ts
 if (this._radios) {
    this._radios.forEach(radio => radio._groupValueChanged());
  }
```
Something like `_markRadiosForCheck`